docs/faq.md

```diff
@@ -26,12 +26,15 @@ in [pod_action.go](https://github.com/heptio/velero/blob/master/pkg/restore/pod_
 
 ## I'm using Velero in multiple clusters. Should I use the same bucket to store all of my backups?
 
-We **strongly** recommend that you use a separate bucket per cluster to store backups. Sharing a bucket
-across multiple Velero instances can lead to numerous problems - failed backups, overwritten backups,
-inadvertently deleted backups, etc., all of which can be avoided by using a separate bucket per Velero
-instance.
+We **strongly** recommend that each Velero instance use a distinct bucket/prefix combination to store backups.
```

Comment:
I think this could be clarified that a bucket/prefix can have multiple readers (so read-only BSLs or a server in restore mode), but only 1 writer.

Reply:
I did some reorganization of the response - this is covered a little more explicitly in the third paragraph now.